modules/Administration/language/en_us.lang.php

```diff
@@ -839,7 +839,7 @@
     'LBL_AOP_JOOMLA_ACCOUNT_CREATION_EMAIL_TEMPLATE' => 'Joomla Support Portal Account Creation Template',
 
     'LBL_AOP_BUSINESS_HOURS_SETTINGS' => 'Business Hours',
-    'LBL_AOP_BUSINESS_HOURS_DESC' => 'Restrict \'Only In The Scheduler\' Workflows to launch only in certain days and times',
+    'LBL_AOP_BUSINESS_HOURS_DESC' => "Restrict 'Only In The Scheduler' Workflows to launch only in certain days and times",
```

Comment:
Isn't the value test to be enclosed into single quotes, not double?
I think the most fashioned way is to use
'LBL_LABEL' => 'text "special" more text'

Reply:
I'll have to test it, but if I want to show single quotes (apostrophes) on the screen, it should be possible...

Reply:
Tested, works well. I see a lot of other strings in that file with double-quotes. According to the project coding standards, this should be avoided when variable substitution is not occurring inside the string. So you would use:

Reply:
'Text'
"Text with $var"
But inthis case we have a reason to use double-quotes, which is: we want an apostrophe as part of the string.